### ISSUE TITLE
Inventory list

### ISSUE THREAD

**Issue by fraserh** [fraserh/waldi#2]
Two associated problems:

1) Create a store locator where a user inputs a postcode, it displays the closest stores for user to choose from
2) Once both stores are chosen, have filters to pass to the db to only retrieve items avaliable at their stores

**Comment by matthewpalmer** [comment]
How much difference is there between stores? Also are there any postcodes with two or more stores in them? 
We need to do this at some stage for sure, but we might be able to get by without it in the 1.0

**Comment by fraserh** [comment]
Yeah, after thinking about it more, I think this is one we leave until it grows
